include/OpenColorIO/OpenColorIO.h

```diff
@@ -161,6 +161,17 @@ extern OCIOEXPORT void ResetToDefaultLoggingFunction();
 /// Log a message using the library logging function.
 extern OCIOEXPORT void LogMessage(LoggingLevel level, const char * message);
 
+/**
+ * @brief Set the Compute Hash Function object to use; otherwise, use the
+ * default.
+ * 
+ * @param hashFunction 
+ * @return OCIOEXPORT 
+ */
+extern OCIOEXPORT void SetComputeHashFunction(ComputeHashFunction hashFunction);
+extern OCIOEXPORT void ResetComputeHashFunction();
+extern OCIOEXPORT std::string ComputeHash(const std::string & filename);
```

Comment:
There is no need for now to directly access to the `ComputeHash` method so that line must ne removed.

Reply:
Removed it sir.